### PR TITLE
Added support for Windows2019 and Windows2022 as amifamily

### PR DESCRIPTION
**Pull request by ROunofF**
*Issue #, if available:* 

*Description of changes:* Added new values for Windows support of Karpenter : Windows 2019 and Windows2022 with relevant tests.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by elamaran11 — COMMENTED** [review]
LGTM. @youngjeong46 @shapirov103 Please check this

**Review by shapirov103 — APPROVED** [review]
LGTM

File: test/karpenter.test.ts [hunk 1]
```diff
@@ -73,6 +73,30 @@ describe('Unit tests for Karpenter addon', () => {
         }).toThrow("Consolidation and ttlSecondsAfterEmpty must be mutually exclusive.");
     });
 
+    test("Support Windows2019 as amiFamily", () => {
```
Comment (shapirov103) [inline test/karpenter.test.ts:76]: please make sure github warnings are addressed here, I suppose running `make lint` should be enough to get the errors.

File: test/karpenter.test.ts [hunk 2]
```diff
@@ -73,6 +73,30 @@ describe('Unit tests for Karpenter addon', () => {
...
+        const app = new cdk.App();
+
+        const blueprint = blueprints.EksBlueprint.builder();
+
+        blueprint.account("123567891").region('us-west-1')
+            .version("auto")
+            .addOns(new blueprints.KarpenterAddOn({
```
Comment (shapirov103) [inline test/karpenter.test.ts:83]: What is this test case accomplishing, if we never build the blueprint? Do you want to add some expectations here, or at least invoke `build` on the blueprint?

Applies elsewhere.

**Review by shapirov103 — CHANGES_REQUESTED** [review]
@ROunofF please run make lint on this - small things in the test file like app is assigned value but not used. Please see my comment on the test file as well.

**Pull request merged**